README.md

```diff
@@ -1,2 +1,42 @@
-# LeaderBoard
-Leaderboard applicaton | JavaScript
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Leaderboard
+
+> Leader board  application using Js/webpack
+
+
+## Built With
+
+- Js
+- web pack
+
+## Live Demo 
+> 
+
+
+## Getting Started
+This is a simple html/css and js  project.
+
+
+## Authors
+
+👤 **Alazar Yilma**
+
+- GitHub: [@Agetuni](https://github.com/Agetuni)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/aleazar-yilma-b614b6174/)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- Please, kindly add the missing license content as the link is broken to make your readme more professional.